ci/aws-analytical-env/shared/meta.yml

```diff
@@ -87,13 +87,15 @@ meta:
               cp ../../../../terraform-config/terraform.tfvars .
               terraform workspace show
               terraform init
-              terraform apply -auto-approve -var assume_role=ci
```

Comment:
we need to keep the assume_role=ci (for now)

Reply:
not anymore 😄